### PR TITLE
feat: add enhancements to conform dialog component

### PR DESCRIPTION
**Pull request by zikowang**
| Status  | Type  |
| :---: | :---: |
| Ready| Feature |

## Motivation and Context
- use confirm dialog component for delete user
- enhance confirm dialog component

## Screenshots / GIFs (if appropriate):

image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Enhancement (non-breaking change which cleans up / improves existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

**Comment by zikowang** [comment]
@noctera i did some cleanup in profile.scss. Please check it out :)

**Review by noctera — APPROVED** [review]

File: src/screens/Profile/Profile.jsx [hunk 1]
```diff
@@ -51,18 +52,20 @@ const Profile = () => {
...
-    if (value === username) {
-      setCanSubmitDelete(true);
-    } else {
-      setCanSubmitDelete(false);
-    }
-  };
-
-  const handleLogout = useCallback(() => {
-    dispatch(signOut());
+  const onDelete = useCallback(async () => {
```
Comment (luwol03) [inline src/screens/Profile/Profile.jsx:55]: why is this function async?

File: src/screens/Profile/Profile.jsx [hunk 2]
```diff
@@ -134,35 +137,24 @@ const Profile = () => {
       >
         <h1 style={{ margin: "auto" }}>Hello World</h1>
       </Modal>
-      <Modal
+
+      <ConfirmDialog
         title={t("screens.profile.modals.deleteAccount.heading")}
-        open={showDeleteAccountModal}
+        onSubmit={onDelete}
         onClose={closeDeleteAccountModal}
```
Comment (luwol03) [inline src/screens/Profile/Profile.jsx:144]: can you also clear the `deleteConfirmation` value in the `closeDeleteAccountModal` function?

File: src/screens/Profile/Profile.jsx (hunk 1 above)
Comment (zikowang) [reply src/screens/Profile/Profile.jsx]: ist not :P

**Review by luwol03 — APPROVED** [review]

**Pull request merged**